css/side_panel.css

```diff
@@ -0,0 +1,17 @@
+.sp-container {
+  background-color: var(--dark-two);
+  min-width: 280px;
+}
+
+.sp-container .title-card {
+  border-radius: 0px;
+  background-color: var(--dark-three);
+  color: var(--gray-two);
+  margin-right: 10px;
+}
+
+.sp-container .ant-card-body {
+  padding: 12px 0px 16px 21px;
```

Comment:
Just a check, should the card body keep a 0px right border? It seems strange to me that the title card has a defined right margin value while the body has 0px. 

Also totally understandable if this is fixing a weird Ant layout, just thought I'd ask. Maybe add a comment here if so.

Reply:
There is probably a smarter way to do this, but adding padding or margin on the right of `.ant-card-body` does not produce the needed space between the card and the side panel edge that the design shows, adding that margin to `.title-card` does the trick.

Meanwhile `.ant-card-body` has 24px default padding all around, so we're using this rule to shrink it down to the desired size.  Similarly, adding padding to '.title-card' isn't helpful.